### PR TITLE
Task/cor 1063 sewer choropleth enhancements

### PR DESCRIPTION
**Pull request by VWSCoronaDashboard18**
## Summary

Adjusting the GM choropleth for Sewer per RWZI
Added a deselect option in the RWZI dropdown to unselect the active RWZI
And added the weekly average to the tooltip.

#### Before
<details>
<summary>Toggle before screenshots</summary>

Screenshot 2023-03-13 at 11 11 02

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

Screenshot 2023-03-13 at 11 25 54

Screenshot 2023-03-13 at 11 26 01

</details>

#### Before
<details>
<summary>Toggle before screenshots</summary>

Screenshot 2023-03-13 at 11 12 50

Screenshot 2023-03-13 at 11 13 00

</details>

#### After
<details>
<summary>Toggle after screenshots</summary>

Screenshot 2023-03-13 at 11 24 14

Screenshot 2023-03-13 at 11 24 38

Screenshot 2023-03-13 at 11 24 21

Screenshot 2023-03-13 at 11 24 30

</details>

### CODE REVIEW

File: packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx [hunk 1]
```diff
@@ -42,6 +43,11 @@ interface SewerChartProps {
     averagesLegendLabel: string;
     averagesTooltipLabel: string;
     valueAnnotation: string;
+    rwziSelectDropdown?: {
```
Comment (VWSCoronaDashboard26) [inline packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx:46]: Is it desirable that these texts are optional? Wouldn't we rather 'force' some type of text to go with this?

Tying into this, I personally don't think that it is desirable to have an empty string as option for the 'deselect'/'show everything' option. Is there any particular reason why you implemented this?

File: packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx [hunk 2]
```diff
@@ -90,6 +97,14 @@ export const SewerChart = ({ accessibility, dataAverages, dataPerInstallation, t
 
   const scopedWarning = useScopedWarning(vrNameOrGmName || '', warning || '');
 
+  const [timeframe, setTimeframe] = useState<TimeframeOption>(TimeframeOption.ALL);
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard18) [reply packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx]: I adjusted it accordingly. Please resolve!

File: packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx [hunk 3]
```diff
@@ -90,6 +97,14 @@ export const SewerChart = ({ accessibility, dataAverages, dataPerInstallation, t
...
+
+  useEffect(() => {
+    if (setSewerTimeframe) {
+      setSewerTimeframe(timeframe);
+    }
+  }, [timeframe, setSewerTimeframe]);
```
[the remark this answers is not in the document]
Comment (VWSCoronaDashboard28) [reply packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx:104]: +1

[the remark this answers is not in the document]
Comment (VWSCoronaDashboard18) [reply packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx:104]: I adjusted it accordingly. Could you please resolve this if you agree?

File: packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx (hunk 1 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx:46]: The reason for the text not being optional is that it's still being used on the national level.
If you agree or have a better solution. Please share or resolve this comment.

**Review by VWSCoronaDashboard28 — DISMISSED** [review]

File: packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx [hunk 4]
```diff
@@ -125,25 +138,33 @@ export const SewerChart = ({ accessibility, dataAverages, dataPerInstallation, t
...
-          <RichContentSelect
-            label={text.selectPlaceholder}
-            visuallyHiddenLabel
-            initialValue={selectedInstallation}
-            options={optionsWithContent}
-            onChange={(option) => onChange(option.value)}
-          />
-        </Box>
-      )}
+      <Spacer marginBottom={space[4]} />
```
Comment (VWSCoronaDashboard21) [inline packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx:147]: Maybe this can be even `{space[1]}`

**Review by VWSCoronaDashboard28 — APPROVED** [review]

**Review by VWSCoronaDashboard26 — APPROVED** [review]

File: packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx (hunk 4 above)
Comment (VWSCoronaDashboard18) [reply packages/app/src/domain/sewer/sewer-chart/sewer-chart.tsx]: I updated the spacing and design gave approval. Therefor I am resolving this comment.

**Pull request merged**